internal/server/stat_series.go

```diff
@@ -120,6 +121,12 @@ func (s *Server) GetStats(ctx context.Context, in *pb.GetStatsRequest) (
 	placeDcids := in.GetPlace()
 	statsVarDcid := in.GetStatsVar()
 
+	log.Printf(
+		"GetStats(): placeDcids: %s, statsVarDcid: %v",
+		placeDcids,
+		statsVarDcid,
+	)
```

Comment:
How will this show up in the log? Is there any way to line it up with the request time (aka, how to tell a request with these args took N secs)?

If the requests generally succeeded, an option is to log the stats with latency when we return.

Reply:
Logged the processing time as well (it would be less than latency though) when it's > 30s.

Reply:
Nice.  If this works ok, we can probably add this to all APIs.